src/config/redis.ts

```diff
@@ -56,7 +56,10 @@ const DataTypeTransforms: { [key: string]: (arg0: any) => any } = {
  * Public transform function
  */
 
-const TransformValueForKey = (key: string, value: string | null) => {
+const TransformValueForKey = async (
+  key: string,
+  value: string | null
+): Promise<any> => {
```

Comment:
```suggestion
): Promise<string | null> => {
```

let's try to avoid introducing more `any`s :)

Reply:
This method returns different types depending on the transform.